sound/soc/sof/topology.c

```diff
@@ -1531,10 +1531,9 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 	/* check token parsing reply */
 	if (ret < 0) {
 		dev_err(scomp->dev,
-			"error: failed to add widget id %d type %d name : %s stream %s\n",
-			tw->shift, swidget->id, tw->name,
-			strnlen(tw->sname, SNDRV_CTL_ELEM_ID_NAME_MAXLEN) > 0
-				? tw->sname : "none");
+			"error: failed to add widget type %d name : %s stream %s\n",
```

Comment:
while we're at it, lets also remove the error prefix in the message and also in the commit message can you mention that the widget id is incorrect and thats why its not useful